EIPS/eip-7xxx.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 7xxx
+title: New opcode BURN
+description: An opcode to burn native token at the given address
+author: Dev Bear (@itsdevbear)
+discussions-to: <URL>
```

Comment:
Please create a discussions topic on Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5